react-components/src/hooks/network/getAssetsList.ts

```diff
@@ -24,6 +24,8 @@ export const getAssetsList = async (
 ): Promise<{ items: Asset[]; nextCursor: string | undefined }> => {
   const advancedFilter = buildFilter(query);
 
+  // The `aggregatedProperties` parameter is not yet public, and hidden behind an alpha flag,
+  // which is why we cannot use the sdk.assets.list method directly
   return await sdk
     .post<CursorResponse<Asset[]>>(`/api/v1/projects/${sdk.project}/assets/list`, {
       headers: {
```

Comment:
This is not directly related to the rest of the PR, but I came across it when making tests